# ALL:ALL - Se cambia shared_library por library en libva para permitir compilarlo estaticamente.
--- libva/va/meson.build
+++ libva/va/meson.build
@@ -68,7 +68,7 @@
 
 install_headers(libva_headers, subdir : 'va')
 
-libva = shared_library(
+libva = library(
   'va',
   sources : libva_sources +
             libva_headers +
@@ -121,7 +121,7 @@
 
   install_headers(libva_drm_headers, subdir : 'va')
 
-  libva_drm = shared_library(
+  libva_drm = library(
     'va-drm',
     sources : libva_drm_sources +
               libva_drm_headers +
@@ -172,7 +172,7 @@
 
   deps = [ libdrm_dep, x11_dep, xext_dep, xfixes_dep, x11_xcb_dep, xcb_dep, xcb_dri3_dep, libva_dep ]
 
-  libva_x11 = shared_library(
+  libva_x11 = library(
     'va-x11',
     sources : libva_x11_sources +
               libva_x11_headers +
@@ -210,7 +210,7 @@
 
   deps = [ gl_dep, libva_x11_dep ]
 
-  libva_glx = shared_library(
+  libva_glx = library(
     'va-glx',
     sources : libva_glx_sources +
               libva_glx_headers +
@@ -273,7 +273,7 @@
 
   deps = [ libdrm_dep, wayland_dep, libva_dep ]
 
-  libva_wayland = shared_library(
+  libva_wayland = library(
     'va-wayland',
     sources : libva_wayland_sources +
               libva_wayland_headers +
@@ -310,7 +310,7 @@
 
   install_headers(libva_win32_headers, subdir : 'va')
 
-  libva_win32 = shared_library(
+  libva_win32 = library(
     'va_win32',
     sources : libva_win32_sources +
               libva_win32_headers,
